### PR TITLE
Updated Readme and Changelog. Deleted installer.

### PR DESCRIPTION
**Pull request by Ne1gh-RR**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,34 +6,33 @@
...
 ### Changed
 
-- **BREAKING CHANGE:** `ResourceInfoMgr` now accepts wildcards instead of directory names. For example, `ResourceInfoMgr.GetResourcePaths("Graphics/Trees");` should now be replaced with `ResourceInfoMgr.GetResourcePaths("Graphics/Trees/*");`
-- **BREAKING CHANGE:** Renamed `GetSafeNormalize()` to `SafeNormalize()`.
-- **BREAKING CHANGE:** Removed instances of `Width` and `Height` in `Sprite`, `Frame`, `WindowMgr`, `Camera`, `Surface`, and replaced them with `Size`.
-- **BREAKING CHANGE:** Changed boolean `isOutline` to `ShapeFill` enum for shapes.
+- Migrated to NET8.
+- Updated MonoGame to `3.8.2`.
+- Updated Nopipeline to `2.3.0`.
+- **BREAKING CHANGE:** Locked OpenGL on Windows.
```
Comment (Martenfur) [inline CHANGELOG.md]: "Removed Linux and Mac support form DesktopGL target. You will have to implement abstractions interfaces to make it work again."

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,34 +6,33 @@
...
+- **BREAKING CHANGE:** Replaced `MonofoxePlatform` class with the Windows implementation.
+- **BREAKING CHANGE:** Replaced `TextInputBinderDesktopGL` class with the Windows implementation.
+- **BREAKING CHANGE:** Replaced `AlphaBlendEffectLoaderWindowsGL` class with the Windows implementation.
 
 ### Fixed
 
-- Fixed `AddComponent<>()` not taking generic type into account.
-- `DirectoryResourceBox` now ignores non-xnb files properly.
-- Fixed division by zero in dampers.
+- Small typos.
```
Comment (Martenfur) [inline CHANGELOG.md]: Not significant enough to be included in a changelog.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,34 +6,33 @@
...
 
 ### Removed
-
-- **BREAKING CHANGE:** Removed camera layer filters.
-- **BREAKING CHANGE:** Removed `Drawable` class and non-static shape fields and methods.
+- **BREAKING CHANGE:** Removed `AlphaBlendEffectLoaderDesktopGl` class.
+- **BREAKING CHANGE:** Removed `MonofoxePlatform` class.
```
Comment (Martenfur) [inline CHANGELOG.md]: Same thing twice.

File: README.md [hunk 4]
```diff
@@ -16,39 +16,38 @@ Monofoxe took a lot of inspiration from Game Maker, so it should be a bit famili
 
 # Getting started
 
-- Download and install [Visual Studio](https://visualstudio.microsoft.com/) 2019 or 2022. 
-- Download `MonofoxeSetup.exe` from the [latest release](https://github.com/Martenfur/Monofoxe/releases/latest). Installer bundles Visual Studio 2019 and 2022 templates. Monofoxe is distributed via nugets, so templates aren't mandatory. It's also a good idea to install project templates for [Monogame](https://github.com/MonoGame/MonoGame/releases/latest).
-
+- Download and install [Visual Studio](https://visualstudio.microsoft.com/) 2022 (VS2019 is no longer supported).
```
Comment (Martenfur) [inline README.md]: VS2019 is no longer supported is not needed.

File: README.md [hunk 5]
```diff
@@ -57,13 +56,13 @@ Yes, you can. Monofoxe is licensed under MIT, so you can use it and its code in
 
 # Should I use it?
 
-Well, up to you. Currently I am developing Monofoxe alone, and can't really provide huge support, or anything. This is mostly an engine for myself and my games - I am not naive enough to think, that everyone will suddenly rush and drop Unity/Game Maker/whatever in favor of Monofoxe. But you can try. ( - :
+yes
 
 # I've suddenly started loving foxes and want to contribute
 
-That's the spirit. Check out if I need any help on my [Quire board](https://quire.io/w/Monofoxe/?board=Monofoxe). Stuff under `Open for taking` category is, well, open for taking. You can also contact me via email (`chaifoxes@gmail.com`), on [Twitter](https://twitter.com/ChaiFoxes) or on [Discord](https://discord.gg/F9tPYaD).
+That's the spirit, but **don't forget to check out Codestyle.cs before contributing!**
```
Comment (Martenfur) [inline README.md]: wtf why remove all the links

File: README.md [hunk 6]
```diff
@@ -16,39 +16,38 @@ Monofoxe took a lot of inspiration from Game Maker, so it should be a bit famili
...
 - Make sure you have either WindowsDX or DesktopGL project selected as default one, and you're good to go!
 
-You can also check out the [basic feature demos](Samples/) or the [Docs](Docs/README.md) to learn how to use Monofoxe.
+You can also check out the [basic feature demos](Samples/) or the [Docs](Docs/README.md) to learn the basics of Monofoxe.
 
 # What can it do?
 
 Everything Monogame does, plus:
 
-* Graphics pipeline and automated batch\vertex buffer management.
```
Comment (Martenfur) [inline README.md]: Did you just swap their places around? Why?

File: Installer/packInstaller.nsi
Comment (Martenfur) [inline Installer/packInstaller.nsi:1]: Bring these back. Installers are needed as for long as we don't have the alternative. Remove them when dotnet templates will be added.

**Review by Martenfur — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (Ne1gh-RR) [reply README.md]: I brought some meaning in here. From the upper text, it's logical to assume that the engine should be simple for newbies in gamedev. This list primarily covers things that newbies usually don't know.

File: Installer/packInstaller.nsi
Comment (Ne1gh-RR) [reply Installer/packInstaller.nsi:1]: kk

File: README.md (hunk 5 above)
Comment (Ne1gh-RR) [reply README.md]: Quire board isn't relevant anymore? 
idk about twitter

File: README.md (hunk 6 above)
Comment (Ne1gh-RR) [reply README.md]: Keep it pls.

**Pull request closed without merging**